scheduler/pkg/agent/client_test.go

```diff
@@ -393,7 +393,7 @@ func TestLoadModel(t *testing.T) {
 			time.Sleep(50 * time.Millisecond)
 
 			// Do the actual function call that is being tested
-			err := client.LoadModel(test.op)
+			err := client.LoadModel(test.op, time.Now().Unix())
```

Comment:
Probably doesn't matter for the test, but for the test to match the actual code, could we also use `time.Now().UnixMilli()`? Or was there another reason why the LoadModel is passed seconds in the test (here and below) rather than milliseconds?

Reply:
If it was just because in the test we don't need that fine of a time resolution, perhaps that's just fine, but wanted to check

Reply:
yes in the tests we can even have handcrafted ts and not necessarily unix ticks. I will actually do that to make it clear that it doesnt have to be unix ticks.

Reply:
done in https://github.com/SeldonIO/seldon-core/pull/5969/commits/c90bc4a855ba1058d32fbb2298a133e00e3aaca2